installer/roles/kubernetes/templates/configmap.yml.j2

```diff
@@ -205,3 +205,5 @@ data:
     USE_X_FORWARDED_PORT = True
 
     AWX_CONTAINER_GROUP_DEFAULT_IMAGE = "{{ container_groups_image }}"
+    BROADCAST_WEBSOCKETS_PORT = 8052
+    BROADCAST_WEBSOCKETS_PROTOCOL = 'http'
```

Comment:
As discussed, before we call this *completely* done, we should come up with some way to put this behind https.